resources/Geometry/BoomBox.gltf

```diff
@@ -0,0 +1,172 @@
+{
+  "accessors": [
+    {
+      "bufferView": 0,
+      "componentType": 5126,
+      "count": 3575,
+      "type": "VEC2"
+    },
+    {
+      "bufferView": 1,
+      "componentType": 5126,
+      "count": 3575,
+      "type": "VEC3"
+    },
+    {
+      "bufferView": 2,
+      "componentType": 5126,
+      "count": 3575,
+      "type": "VEC4"
+    },
+    {
+      "bufferView": 3,
+      "componentType": 5126,
+      "count": 3575,
+      "type": "VEC3",
+      "max": [
+        0.009921154,
+        0.00977163,
+        0.0100762453
+      ],
+      "min": [
+        -0.009921154,
+        -0.00977163,
+        -0.0100762453
+      ]
+    },
+    {
+      "bufferView": 4,
+      "componentType": 5123,
+      "count": 18108,
+      "type": "SCALAR"
+    }
+  ],
+  "asset": {
+    "generator": "glTF Tools for Unity",
+    "version": "2.0"
+  },
+  "bufferViews": [
+    {
+      "buffer": 0,
+      "byteLength": 28600
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 28600,
+      "byteLength": 42900
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 71500,
+      "byteLength": 57200
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 128700,
+      "byteLength": 42900
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 171600,
+      "byteLength": 36216
+    }
+  ],
+  "buffers": [
+    {
+      "uri": "BoomBox.bin",
+      "byteLength": 207816
+    }
+  ],
+  "images": [
+    {
+      "uri": "BoomBox_baseColor.png"
```

Comment:
One minor concern I have is that this `*.gltf` file is not valid as presented here, because the images associated with the URIs here are all in the `Images` folder.  I realize the whole structure of this project keeps images away from geometry, but that's not going to allow most glTF loaders to parse this as-is.  Relative paths are allowed, although paths reaching into parent folders are strongly discouraged, as some implementations reject them for security.

Reply:
I've moved the gltf file and images to be co-located so the references should be okay now.
This should hopefully also make it so that there is no additional packing step for resources for usage by the MaterialXView web viewer.
-----
As part of this @jstone-lucasfilm , I put in new logic for resolving relative pathing for OSL render tests. I removed the current patch logic as such. As part of this I wrote a simple utility to remove ".." qualifiers embedded in `FilePath`s which testrender and the MDL dxr test renderer don't handle.